docker/README.md

```diff
@@ -0,0 +1,32 @@
+# RackHD on Docker instructions
+
+Copyright 2016, EMC, Inc.
+
+Install Docker Toolbox on your system. See https://www.docker.com/products/docker-toolbox
+
+This has only been tested on Mac OS X.
```

Comment:
I should update that. It will work on linux but the RackHD repo must be at the root. I'll change the readme.